### ISSUE TITLE
unsatisfied model generated for object mainly in white color

### ISSUE THREAD

**Issue by 2646044472** [TencentARC/InstantMesh#54]
It seems that objects predominantly colored in white, such as toilet paper, are not detected effectively, possibly due to the model being trained primarily with images featuring white backgrounds. Conversely, the model performs well with objects that are not primarily white. Could you suggest any potential solutions to address this issue?
 
![image](https://github.com/TencentARC/InstantMesh/assets/104624750/778c24fa-1b95-44b7-950f-ca5f6a98a364)
![image](https://github.com/TencentARC/InstantMesh/assets/104624750/627705f4-c10d-4fa6-8577-731bba9ce12e)
![image](https://github.com/TencentARC/InstantMesh/assets/104624750/67340050-983a-4312-b33d-3c42d35272a2)

**Comment by bluestyle97** [comment]
Yes, this is indeed a problem of using white background. The original Zero123++ generates gray background, which will meet the similar problem if the image foreground contains gray-color area in my opinion. By far we cannot do much to this issue, maybe you can try using some image processing tools or image-to-image style translation generative models to make the highlight area in the input image darker. We will try to fix this issue in the next version.
